apps/ringtones/js/actions_menu.js

```diff
@@ -139,7 +139,7 @@ ActionsMenu.prototype = {
     };
     CustomDialog.show(
       'delete-title',
-      {id: descKey, args: {tone: self._tone.name}},
+      {id: messageKey, args: {tone: self._tone.name}},
```

Comment:
No. If the string in the l10n file isn't HTML (read: has neither a `<` nor a `&`), then everything is assumed to be plaintext. I'd imagine the args themselves are *always* treated as plain text, since it'd certainly be strange if a different l10n string caused the arguments to be parsed differently.

Even replacing `<bdi>` and `</bdi>` with the corresponding Unicode codepoints doesn't work, probably because the W3C spec explicitly says that HTML parsers should ignore those codepoints (they argue that it's harder to use them than to use `<bdi>` and `</bdi>`).